sbt-crossproject/src/main/scala/sbtcrossproject/CrossProject.scala

```diff
@@ -208,26 +209,38 @@ object CrossProject {
     private def sharedSrcSettings(crossType: CrossType): Seq[Setting[_]] = {
       def makeCrossSources(sharedSrcDir: Option[File],
                            scalaBinaryVersion: String,
+                           scalaEpochVersion: String,
                            cross: Boolean): Seq[File] = {
         sharedSrcDir match {
           case Some(dir) =>
             if (cross)
-              Seq(dir.getParentFile / s"${dir.name}-$scalaBinaryVersion", dir)
+              Seq(
+                dir.getParentFile / s"${dir.name}-$scalaBinaryVersion",
+                dir.getParentFile / s"${dir.name}-$scalaEpochVersion",
+                dir
+              )
             else
               Seq(dir)
```

Comment:
Since there is now one branch with more than one line, our coding style asks for braces:
```suggestion
            if (cross) {
              Seq(
                dir.getParentFile / s"${dir.name}-$scalaBinaryVersion",
                dir.getParentFile / s"${dir.name}-$scalaEpochVersion",
                dir
              )
            } else {
              Seq(dir)
            }

Reply:
I ran scalafmt but it seems to be missing this rule.

Reply:
scalafmt cannot do everything. This is one of the things we cannot tell it to do.